charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -56,19 +56,34 @@ spec:
             - name: KEYCLOAK_ADMIN
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  {{- if .Values.auth.existingSecret }}
+                  name: {{ .Values.auth.existingSecret }}
+                  {{- end }}
+                  {{- if not .Values.auth.existingSecret }}
+                  name: {{ .Values.auth.name }}
```

Comment:
In the secret template, the name is set to .Release.Name, not .Values.auth.name

Reply:
Fixed that.